GiniCapture/Classes/Core/Logging/ErrorLog.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  ErrorLog.swift
+//  GiniCapture
+//
+//  Created by Nadya Karaban on 20.07.21.
+//
+
+import Foundation
+import GiniPayApiLib
+public struct ErrorLog {
```

Comment:
Isn't `ErrorLog: Codable` enough without the need for `enum CodingKeys` and the `Decodable` extenstion? The property names are the same.

Reply:
ithink it's needed when you don't want to decode /encode everything and for nested types